### PR TITLE
[NEW] wegenbestand dataset

### PR DESCRIPTION
**Pull request by cvriel**
The wegenbestand contains data about so called `zones zwaar verkeer` en consists of roads within that zone, outside that zone and roads that needs a special waiver to be used. These three tables all fall within the main category wegenbestand and subcategory zones zwaar verkeer.

The existing datasets hoofdroutes, like `routes and tunnels gevaarlijke stoffen` are also placed on the `wegenbestand` but not under `zones zwaar verkeer`. They reside on the main level in that perspective.

The DAG hoofdroutes will be for now still active. In a later stadium that DAG will be de-allocated.

### CODE REVIEW

**Review by jjmurre — APPROVED** [review]
LGTM (only some small typo's)

File: src/dags/wegenbestand.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    Interface = DummyOperator(task_id="interface")
+
+    # 5a. path 1: import geopackage to database
+    load_geopackage = [
+        Ogr2OgrOperator(
+            task_id=f"import_gpkg_layer_{values['table']}",
+            sql_statement="SELECT *"  # noqa: S608
+            f" FROM {quote_string(values['geopackage_layer'])}"
+            f" WHERE zone_zzv = {quote_string(values['data_filter'])}",
+            input_file=f"{TMP_DIR}/{resource_name}.{extention['file_type']}",
```
[the remark this answers is not in the document]
Comment (cvriel) [reply src/dags/wegenbestand.py:91]: :+1:

File: src/dags/wegenbestand.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+for (geo_valid, multi_check, capture_data, clean_up) in zip(
+    make_geo_valid, multi_checks, change_data_capture, clean_ups
+):
+    [geo_valid >> multi_check >> capture_data >> clean_up]
+
+clean_ups >> grant_db_permissions
+
+dag.doc_md = """
+    #### DAG summary
+    This DAG contains data about city accesibility routes/zones for heavy classified traffic.
```
[the remark this answers is not in the document]
Comment (cvriel) [reply src/dags/wegenbestand.py:268]: :+1:

File: src/dags/wegenbestand.py [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    #### Mission Critical
+    Classified as 2 (beschikbaarheid [range: 1,2,3])
+    #### On Failure Actions
+    Fix issues and rerun dag on working days
+    #### Point of Contact
+    Inform the businessowner (Bas Bussink) at bereikbaarheidsthermometer@amsterdam.nl
```
[the remark this answers is not in the document]
Comment (cvriel) [reply src/dags/wegenbestand.py:274]: yeay!

**Pull request merged**